src/databot.js

```diff
@@ -0,0 +1,19 @@
+const nqmUtils = require("@nqminds/nqm-core-utils");
+
+async function abortDatabot(api, id) {
+  return api.abortDatabotInstance(id);
```

Comment:
These functions don't need to be async if api is an instance of nqm-api-tdx as they will return promises by default.